components/hidden_data_box/commons/hidden_data_box.lua

```diff
@@ -31,71 +35,111 @@ function HiddenDataBox.run(args)
 		= HiddenDataBox.validateTier(args.liquipediatiertype, TIER_MODE_TYPES)
 	table.insert(warnings, warning)
 
-	local parent = args.tournament or tostring(mw.title.getCurrentTitle().basePageTitle)
+	local parent = args.parent or args.tournament or tostring(mw.title.getCurrentTitle().basePageTitle)
 	parent = parent:gsub(' ', '_')
 
 	local queryResult = mw.ext.LiquipediaDB.lpdb('tournament', {
 		conditions = '[[pagename::' .. parent .. ']]',
 		limit = 1,
-		order = 'pagename asc',
 	})
-	queryResult = queryResult[1] or {}
-
-	local startDate = HiddenDataBox:cleanDate(args.sdate, args.date)
-	local endDate = HiddenDataBox:cleanDate(args.edate, args.date)
-	HiddenDataBox:checkAndAssign('tournament_startdate', startDate, queryResult.startdate)
-	HiddenDataBox:checkAndAssign('tournament_enddate', endDate, queryResult.enddate)
-
-	HiddenDataBox:checkAndAssign('tournament_name', args.name, queryResult.name)
-	HiddenDataBox:checkAndAssign('tournament_series', args.series, queryResult.series)
-	HiddenDataBox:checkAndAssign('tournament_shortname', args.shortname, queryResult.shortname)
-	HiddenDataBox:checkAndAssign('tournament_tickername', args.tickername, queryResult.tickername)
-	HiddenDataBox:checkAndAssign('tournament_icon', args.icon, queryResult.icon)
-	HiddenDataBox:checkAndAssign('tournament_icondark', args.icondark or args.icondarkmode, queryResult.icondark)
-	HiddenDataBox:checkAndAssign('tournament_liquipediatier', args.liquipediatier, queryResult.liquipediatier)
-	HiddenDataBox:checkAndAssign('tournament_liquipediatiertype', args.liquipediatiertype, queryResult.liquipediatiertype)
-	HiddenDataBox:checkAndAssign('tournament_type', args.type, queryResult.type)
-	HiddenDataBox:checkAndAssign('tournament_status', args.status, queryResult.status)
-	HiddenDataBox:checkAndAssign('tournament_game', args.game, queryResult.game)
-	HiddenDataBox:checkAndAssign('tournament_parent', args.parent, parent)
-	HiddenDataBox:checkAndAssign('tournament_parentname', args.parentname, queryResult.name)
-
-	HiddenDataBox:addCustomVariables(args, queryResult)
+	queryResult = queryResult[1]
+
+	if not queryResult then
+		table.insert(warnings, String.interpolate(INVALID_PARENT, {parent = parent}))
+		queryResult = {}
+	elseif args.participantGrabber then
```

Comment:
Logic.readBool
else there will be no option to disable it (as `'false'` is treated as true here since it is a string and not a bool)